app-web/cinny-desktop/autobuild/beyond

```diff
@@ -0,0 +1,8 @@
+abinfo "Installing application icon ..."
+for i in 32 128; do
+    install -Dvm644 "$SRCDIR"/icons/${i}x${i}.png \
+        "$PKGDIR"/usr/share/icons/hicolor/${i}x${i}/apps/cinny.png
+done
+
+install -Dvm644 "$SRCDIR"/icons/icon.png \
```

Comment:
This is probably not necessary (also having the icon installed as `icon.png` in that path is probably not wise).

If you want (since it is indeed a higher-resolution icon), install it to the `512x512` directory.